.github/workflows/tests.yml

```
@@ -19,6 +19,7 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.7', '3.8', '3.9', '3.10']
+        use-lower-version: [true, false]
```

The CI job name will be `Tests (3.7, false)` something like that. It is difficult to see the actual job. How about using the string as follows?
```suggestion
        use-lower-version: ['', 'use-lower-version']
```